src/helpers/FilterPosts.js

```diff
@@ -0,0 +1,30 @@
+// Put a proper readme
+// explaining about the app, what you’re using
+// your file structure and setup instructions PR description on what is there in here
+// And follow a commit discipline like this https://zulip.readthedocs.io/en/latest/contributing/version-control.html
+// Relative time (1 day ago or 2 hrs ago etc)
+
+import { db } from "../firebaseConfig"
```

Comment:
Why separate file? Group helpers together. Having one file for one function doesn't make sense.